containers/codespaces-linux/.devcontainer/library-scripts/common-debian.sh

```diff
@@ -107,7 +110,10 @@ if [ "${PACKAGES_ALREADY_INSTALLED}" != "true" ]; then
         sudo \
         ncdu \
         man-db \
-        strace"
+        strace \
+        manpages-dev \
```

Comment:
Maybe add "manpages" as well?  This should get pulled in by the others, but worth calling it out explicitly IMHO.

Reply:
Added !